pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -572,18 +698,22 @@ func (vs *VSphere) DisksAreAttached(volPaths []string, nodeName k8stypes.NodeNam
 		if len(volPaths) == 0 {
 			return attached, nil
 		}
-		var vSphereInstance string
```

Comment:
Can you please get the latest code. DisksAreAttached() is completely rewritten for bulk verification of volumes.

Reply:
I'll talk to you about this.